DSMRloggerAPI.ino

```diff
@@ -110,7 +114,7 @@ void setup()
   Serial.printf("\n\nBooting....[%s]\r\n\r\n", String(_FW_VERSION).c_str());
 
 #if defined( HAS_OLED_SSD1306 ) || defined( HAS_OLED_SH1106 )
-  oledSleepTimer = millis() + (10 * 60000); // initially 10 minutes on
+  DECLARE_TIMER_MIN(oledSleepTimer, 10);
```

Comment:
Door de timer hier te definiëren kun je hem nooit meer aanpassen als de settings worden aangepast.
Beter: in API.h declareren.